### DISCUSSION TITLE
How to display log entries sent via network?

### DISCUSSION THREAD

**Discussion by vissjb** [boexler/NLogViewer#78] [Q&A]
In a client/server scenario, where the server is sending it's log events via a Network or NLogViewer target, how would you display them in the client application?

**Comment by boexler** [comment]
Hi @vissjb ,
Sorry for the late reply. I didn't see that a discussion had been started here.
This should be fixed by https://github.com/djonasdev/NLogViewer/issues/41. I plan to start working on it soon.
